### ISSUE TITLE
Fundamental research/AI/NLP projects

### ISSUE THREAD

**Issue by LifeIsStrange** [nayafia/microgrants#70]
So I am understanding it right that the most important fields of research have not microgrants funding?

AI grant is (was?) dead https://aigrant.org/

NLNet Foundation seemed great except it require to open source the code which is a no-no... (why would I work on something groundbreaking and get zero economic and executive reward on it?) 

Maybe scifounders is fit for the purpose?
Hope it doesn't require a diploma/license nor physical presence. 

Basically that issue is about the need of including more grants for fundamental research (could be medecine too), that does not require open sourcing work.

**Comment by matkoniecz** [comment]
> why would I work on something groundbreaking and get zero economic and executive reward on it?

Getting grant is a nonzero economic reward (and there are some reasons why open source may be preferred)

> So I am understanding it right that the most important fields of research have not microgrants funding?

If you find something, feel free to open issue. But this issue seems to be complaint that grant is not existing, and therefore not actionable here. It is "A list of microgrant programs" not "List of requested microgrant programs"

**Comment by matkoniecz** [comment]
BTW, there is

> If you're interested in starting your own microgrant program, I've also created [a few templates](https://github.com/nayafia/microgrants/blob/master/templates), which you're welcome to use.

in the readme.

**Comment by LifeIsStrange** [comment]
> Getting grant is a nonzero economic reward
It is not non-zero and that's fine for many projects, but for long term high risk projects that is often unsufficent. 

I just wanted to show visibility for this lack of grants (well there is at least one). Thanks for answering.
